mobile/squads/squad_members.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import { View, Text, FlatList} from "react-native";
+import { squad_members_styles } from "./squad_members_styles";
+import {callBackend } from "../backend/backend"
+
+type Users = {
```

Comment:
imo lets not define a separate type for Users. We should keep types consistent otherwise it can start to get really messy throughout the code. Maybe move User from `login.tsx` into a shared `types/user.tsx` file and we can just use that one type throughout the code